manifests/config.pp

```diff
@@ -110,6 +105,8 @@
   }
 
   if $foreman::apache  {
+    User[$apache::user] -> Systemd::Dropin_file['foreman-socket']
```

Comment:
The line at https://github.com/theforeman/puppet-foreman/pull/883/files#diff-d6542e555b7c0207c63c969d650e1036R5 always ends up blank when I run this and so I was attempting to define a relationship that would ensure it. This did not seem to work either.